scripts/dev/upstream_merge/repos.conf

```diff
@@ -1,14 +1,14 @@
 #local_repo                  upstream_repo                                         upstream_branch       local_base_branch
-sources/bitbake              https://git.openembedded.org/bitbake                  master                nilrt/master/next
+sources/bitbake              https://git.openembedded.org/bitbake                  2.8                   nilrt/master/next
 sources/meta-cloud-services  https://git.yoctoproject.org/meta-cloud-services      master                nilrt/master/next
-sources/meta-openembedded    https://git.openembedded.org/meta-openembedded        master                nilrt/master/next
-sources/meta-selinux         https://git.yoctoproject.org/meta-selinux             master                nilrt/master/next
-sources/openembedded-core    https://git.openembedded.org/openembedded-core        master                nilrt/master/next
+sources/meta-openembedded    https://git.openembedded.org/meta-openembedded        scarthgap             nilrt/master/next
+sources/meta-selinux         https://git.yoctoproject.org/meta-selinux             scarthgap             nilrt/master/next
+sources/openembedded-core    https://git.openembedded.org/openembedded-core        scarthgap             nilrt/master/next
 sources/meta-virtualization  https://git.yoctoproject.org/meta-virtualization      master                nilrt/master/next
```

Comment:
FWIW meta-virtualization also just sprouted a 'scarthgap' branch. It's probably worth checking the other ones below too.

Reply:
Thanks for pointing this out. They must have merged `master-next` to `master` and created `scarthgap` branch today sometime after I ran `upstream_merge.sh` as there are newer commits in upstream's `master` than when I ran the script.